### PR TITLE
Add caluse->clause to dictionary.txt

### PR DESCRIPTION
**Pull request by josh11b**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4848,6 +4848,7 @@ calulates->calculates
 calulating->calculating
 calulation->calculation
 calulations->calculations
+caluse->clause
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:4851]: also `callus` + `callous`

Comment (josh11b) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4848,7 +4848,7 @@ calulates->calculates
 calulating->calculating
 calulation->calculation
 calulations->calculations
-caluse->clause
+caluse->clause,callus,callous,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:4851]: space after the comma is necessary

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:4851]: Like so @josh11b :
```suggestion
caluse->clause, callus, callous,
```

**Review by peternewman — APPROVED** [review]

**Pull request merged**